model/error/_meta/fields.yml

```diff
@@ -4,16 +4,16 @@
   fields:
     - name: view errors
       type: keyword
-      script: "doc['service.name'].size() > 0 ? doc['service.name'].value : doc['context.service.name'].size() > 0 ? doc['context.service.name'].value : null"
+      script: "doc['service.name'].size() > 0 ? doc['service.name'].value : doc['service.name'].size() > 0 ? doc['service.name'].value : null"
```

Comment:
the second set is for compatibility with 6.x indexes and shouldn't change

Reply:
You are right, that needs more thoughts, I am going to revert the dashboard changes here, as they are unrelated to the `user` field changes anyways.